packages/language-server/src/utils/files/resolveTemplate.ts

```diff
@@ -0,0 +1,41 @@
+import { fileStat } from './fileStat';
+
+// HACK: Maybe we should move these to a configuration setting?
+const extensions = [`.twig`, `.html`];
+
+const indexTemplateFilenames = [`index`];
+
+/**
+ * Searches for a template file, and returns the first match if there is one.
+ */
+export const resolveTemplate = async (
+    pathToTwig: string,
+): Promise<string | null> => {
+    let stats = await fileStat(pathToTwig);
+
+    if (stats && stats.isFile()) {
+        return pathToTwig;
+    }
+
+    for (const extension of extensions) {
+        const testPath = `${pathToTwig}${extension}`;
+        stats = await fileStat(testPath);
```

Comment:
nit: Personally I prefer declaring all my variables as `const`, and assigning other names to new variables.

How about we give this variable a unique name, such as `const testPathStats`?